docs/source/api/ptycho.solvers.rst

```diff
@@ -0,0 +1,14 @@
+solvers
+-------
+.. automodule:: tike.ptycho.solvers
+   :inherited-members:
+   :members:
+   :show-inheritance:
+   :undoc-members:
+
+   .. autosummary::
+      :nosignatures:
+
+      admm
+      combined
+      divided
```

Comment:
We should think about better names for those.

Reply:
Agreed. I'm open to suggestions. They all use conjugate gradient, line search, and maximum-likelihood cost functions, so we can't use any of those words. The only way to distinguish them is how the ptychography problem is divided into subproblems and how those subproblems are linked together. I guess one of them could be called 'sequential'? Or we could name them after the authors like we are doing with position correction algorithms?